testing/e2e/skr-tester/pkg/command/assert.go

```diff
@@ -48,12 +55,16 @@ func NewAsertCmd() *cobra.Command {
 	cobraCmd.Flags().StringVarP(&cmd.clusterOIDCConfig, "clusterOIDCConfig", "o", "", "clusterOIDCConfig of the specific instance.")
 	cobraCmd.Flags().StringSliceVarP(&cmd.kubeconfigOIDCConfig, "kubeconfigOIDCConfig", "k", nil, "kubeconfigOIDCConfig of the specific instance. Pass the issuerURL and clientID in the format issuerURL,clientID")
 	cobraCmd.Flags().StringSliceVarP(&cmd.admins, "admins", "a", nil, "Admins of the specific instance.")
+	cobraCmd.Flags().BoolVarP(&cmd.btpManagerSecretExists, "btpManagerSecretExists", "b", false, "Checks if the BTP manager secret exists in the instance.")
+	cobraCmd.Flags().BoolVarP(&cmd.editBtpManagerSecret, "editBtpManagerSecret", "e", false, "Edits the BTP manager secret in the instance and checks if the secret is reconciled.")
+	cobraCmd.Flags().BoolVarP(&cmd.deleteBtpManagerSecret, "deleteBtpManagerSecret", "d", false, "Deletes the BTP manager secret in the instance and checks if the secret is reconciled.")
 
 	return cobraCmd
 }
 
 func (cmd *AssertCommand) Run() error {
 	cmd.log = logger.New()
+	ctrl.SetLogger(zap.New())
```

Comment:
required by controller-runtime, I will try to get rid of it to use slog as stated here https://github.com/kyma-project/kyma-environment-broker/issues/1588#issuecomment-2565442026